modules/signing_worker/manifests/init.pp

```diff
@@ -71,16 +73,20 @@
 
     $required_directories = [
       $scriptworker_base,
-      "${scriptworker_base}/certs",
       "${scriptworker_base}/logs",
-      "${scriptworker_base}/artifact",
     ]
     file { $required_directories:
       ensure => 'directory',
       owner  =>  $user,
       group  =>  $group,
       mode   => '0750',
     }
+    file { "${scriptworker_base}/certs":
+      ensure => 'directory',
+      owner  =>  $user,
+      group  =>  $group,
+      mode   => '0700',
```

Comment:
Since multiple scriptworker processes with different users may be on the box, I'm setting the yaml files and certs directories to user-only perms. (Not the base directory because `poller` is a subdirectory currently, and shares the virtualenv.)